docs/Tutorial_DiscordMERLIN.ipynb

```diff
@@ -0,0 +1,739 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    def _find_discord(subseqs, cands):
</pre><p>For consistency, I think the input should still be <code>is_cands</code>  (the result from _prune_candidates) instead of <code>cands</code>  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
Will be fixed in next commits.